test/runtests.py

```diff
@@ -395,6 +395,59 @@ def on_delete(queue):
                                self.guardian.del_queue_size]
         self.assertTrue(len(queues_to_delete) == 0)
 
+    def test_delete_skip_unbounded(self):
+        self._setup_queue()
+
+        # Queue should still exist.
+        self._wait_for_queue()
+
+        # Get the queue's object
+        db_session.refresh(self.pulse_user)
+
+        self.assertTrue(len(self.pulse_user.queues) > 0)
```

Comment:
Good call.  I fixed this in the rest of the file as well to avoid the dreaded ``copyPastingOfWrongness``  :)